File: Selenium/Project/Activity_2.java
package Activity_CRM;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity_2 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
        WebElement img = driver.findElement(By.xpath("//img[@alt=\"SuiteCRM\"]"));
        String src = img.getAttribute("src");
        System.out.println("url of the header image " +src);
        driver.close();
    }
}

File: Selenium/Project/Activity_2.py
from selenium.webdriver.common.by import By
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("http://alchemy.hguy.co/crm")

   # img = driver.find_element_by_xpath("//img[@alt='SuiteCRM']")
    img = driver.find_element(By.XPATH, "//img[@alt='SuiteCRM']")
    src = img.get_attribute("src")

    print("URL of the header image: " + src)

    driver.quit()